src/HotChocolate/Core/test/Types.Tests/Types/InputObjectTypeAttributeTests.cs

```diff
@@ -96,5 +134,20 @@ public class RenameTypeAttribute
                 descriptor.Name("Bar");
             }
         }
+
+        public class InputWithDefaults
```

Comment:
You could also add an enum property to test. I updated [my issue](https://github.com/ChilliCream/hotchocolate/issues/4220) to include the SDL output for enums with defaults.

Reply:
I have added a tests for this now.